awscli/customizations/sso/utils.py

```diff
@@ -77,6 +77,15 @@ def do_sso_login(session, sso_region, start_url, token_cache=None,
     )
 
 
+def parse_sso_registration_scopes(raw_scopes):
+    parsed_scopes = []
+    for scope in raw_scopes.split(','):
+        scope = scope.strip()
+        if scope:
```

Comment:
Up to you, but a rare opportunity for a walrus!
```suggestion
        if scope := scope.strip():
```

Reply:
Sure. I think this was just copied from over from a previous definition.